Scoop/Supporting/Assets.xcassets/Dots.imageset/Contents.json

```diff
@@ -0,0 +1,12 @@
+{
+  "images" : [
+    {
+      "filename" : "Line 4.pdf",
```

Comment:
why are the images pdf files?

Reply:
Instead of worrying about sizing for three pngs, we can upload one pdf and Xcode will generate all three appropriate png files to fit each resolution. You can read more about it here: https://developer.apple.com/forums/thread/83121

Reply:
:O interesting